app/util.js

```diff
@@ -109,12 +109,19 @@ const searchword = location => {
             const q = location.replace(`${word} `, "")
             if (q && location.startsWith(`${word} `)) {
                 const queries = q.split(",").map(item => item.trim())
-                let i = 0
                 let urlString = url
-                while (urlString.includes("%s")) {
-                    urlString = urlString.replace(/%s/, encodeURIComponent(queries[i]))
-                    i += 1
+                let counter = 1
+                const patternMatches = (urlString.match(/%s/g) || []).length
+                while (urlString.includes("%s") && counter < patternMatches) {
+                    if (0 in queries) {
+                        urlString = urlString.replace(/%s/, encodeURIComponent(queries.shift()))
+                    } else {
+                        urlString = urlString.replace(/%s/g, "")
+                    }
+                    counter += 1
                 }
+                const remainderString = queries.join(", ")
```

Comment:
If you are splitting on `,` I also expect them to be joined with `,` and not with `, `.